peps/pep-0013.rst

```diff
@@ -343,6 +353,7 @@ History of amendments
 ---------------------
 
 2019-04-17: Added the vote length for core devs and changes to this document.
+2024-XX-XX: Adopted Multi-winner Bloc STAR voting for council elections.
```

Comment:
**Remember to fill in the actual date** the PR was approved by a Discord poll (or some other significant date, maybe just the date the PR is merged).

Reply:
you can link to https://discuss.python.org/t/changing-pep-13-to-adopt-bloc-star-voting/64971 here.

Reply:
This suggestion uses a footnote -- I couldn't think of a nice way to incorporate the link other than linking the whole line or having a '(link)' trailer, which we try to avoid.
```suggestion
2024-10-09: Adopted Multi-winner Bloc STAR voting for council elections. [1]_

.. [1]: https://discuss.python.org/t/64971
```

Reply:
I think it's more appropriate for this to link to the vote that approved the change. The changeset/PR can easily be retrieved using GitHub or `git blame`; there's no easy wat to find the voting thread without explicitly linking it here.

Reply:
I've decided to link the word "Adopted" to the discuss.python.org thread with the vote. I am going to merge this now.